### DISCUSSION TITLE
span.recordException not showing it as exception in tracing

### DISCUSSION THREAD

**Discussion by Kibrom1** [open-telemetry/opentelemetry-java#6023] [Q&A]
We have a OpenTelemetry implementation to record our traces over to app insights, Span.recordException doesn't successfully record a Span event as an exception.

**Comment by jack-berg** [comment]
The idea behind `Span.recordException` is that a span event is attached to the span with a very specific shape. See this [specification document](https://github.com/open-telemetry/opentelemetry-specification/blob/13860cb664178141a445589bdbda700e4514655f/specification/trace/exceptions.md) for the expectations of this operation. Do you have an indication that the code from `opentelemetry-java` isn't doing something right? 

It sounds like you may be asking a question about how a particular backend (app insights) handles this type of data. If so, I would recommend asking the question to the folks who run app insights.

**Comment by trask** [comment]
hi @Kibrom1! as @jack-berg mentioned, this is most likely an Application Insights specific question. in particular check out https://github.com/microsoft/ApplicationInsights-Java/issues/2229, and feel free to comment or open a new issue over there, thx
